lib/SPIRV/OCLTypeToSPIRV.h

```diff
@@ -1,3 +1,6 @@
+#ifndef LLVM_TOOLS_SPIRV_LLVM_TRANSLATOR_LIB_SPIRV_OCLTYPETOSPIRV_H
```

Comment:
1. I would prefer move the guards after the copyright header.
2. Should we prefix it with "LLVM_TOOLS_"?

Reply:
> 1. I would prefer move the guards after the copyright header.

I’ll manually edit that.

> 2. Should we prefix it with "LLVM_TOOLS_"?

It was automatically done as I was building in tree, so you get that “LLVM_TOOLS_” prefix. I guess if I was building out of tree, we wouldn’t get it. Looking at the clang headers, they only have an “LLVM_” prefix.